src/relay/backend/vm/compiler.cc

```diff
@@ -1007,6 +998,15 @@ transform::Sequential MemoryOpt(tvm::Target host_target, TargetsMap targets) {
 
 IRModule VMCompiler::OptimizeModule(const IRModule& mod, const TargetsMap& targets,
```

Comment:
We may want to remove `const` on `mod` here.

Reply:
Let's remove the `const` here since we could modify it.